modules/standard/IO.chpl

```diff
@@ -430,7 +430,9 @@ module IO {
       (ie, they can open up channels that are not shared).
 */
 
-public use SysBasic;
+public import SysBasic.{syserr,EFORMAT,fd_t,ENOERR,EEOF,qio_err_t};
+public import OS.POSIX.{ENOENT, ENOSYS, EINVAL, EILSEQ, EIO, ERANGE};
+private import OS.POSIX.{EBADF};
```

Comment:
This list feels fairly arbitrary to me and was based on what passed testing and I found referred to within the sources...  Though it also feels like IO should stop relying on users to access these symbols?  Is there something more principled I could/should do for the time being other than `public use OS.POSIX;` (which I don't want to do).

Reply:
Does IO ever return those symbols?  I would expect us to rely more on throwing SystemError and its subclasses rather than the error codes themselves

Reply:
Checking into this, I didn't find cases where the main user-facing routines returned them, and believe that I'd added the `public imports` originally because I found myself wondering why I was having to update so many other modules and codes after removing the `public SysBasic;`  After yesterday's deep-dive, I changed them back to private and found that most of the cases that had been relying on the `public use SysBasic;` were not exercising user-facing features, so made these `private` again and updated all the other places instead.